frontend/src/component/providers/AccessProvider/AccessProvider.tsx

```diff
@@ -79,7 +78,7 @@ const checkPermission = (
     if (
         p.permission === permission &&
         (p.project === project || p.project === '*') &&
-        p.environment === null
+        p.environment == null
```

Comment:
Why was this change needed? It's generally considered bad practice to allow type coercion in equal checks in javascript.

Reply:
I wanted the check to be for undefined as well

Reply:
I would change it to explicit conversion instead: `Boolean(p.environment) === false`